src/Mod/Part/Gui/TaskFaceAppearances.cpp

```diff
@@ -276,37 +278,43 @@ FaceColors::~FaceColors()
     delete d;
 }
 
-void FaceColors::setupConnections()
+void FaceAppearances::setupConnections()
 {
-    connect(d->ui->colorButton, &Gui::ColorButton::changed,
-            this, &FaceColors::onColorButtonChanged);
     connect(d->ui->defaultButton, &QPushButton::clicked,
-            this, &FaceColors::onDefaultButtonClicked);
+            this, &FaceAppearances::onDefaultButtonClicked);
     connect(d->ui->boxSelection, &QPushButton::toggled,
-            this, &FaceColors::onBoxSelectionToggled);
+            this, &FaceAppearances::onBoxSelectionToggled);
+    connect(d->ui->widgetMaterial,
+            &MatGui::MaterialTreeWidget::materialSelected,
+            this,
+            &FaceAppearances::onMaterialSelected);
+    connect(d->ui->buttonCustomAppearance,
+            &QPushButton::clicked,
+            this,
+            &FaceAppearances::onbuttonCustomAppearanceClicked);

```

Comment:
I'd capitalize the "B" in `onbuttonCustomAppearanceClicked` here, it jumps out as an error to a reader otherwise.